code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1451,6 +1451,29 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 	armor_block = min(90,armor_block) //cap damage reduction at 90%
 	var/Iforce = I.force //to avoid runtimes on the forcesay checks at the bottom. Some items might delete themselves if you drop them. (stunning yourself, ninja swords)
 
+	if(H.mind && H.mind.martial_art && istype(H.mind.martial_art, /datum/martial_art/monk))
```

Comment:
If my pitiful understanding of things is right the problem lays here somewhere for the no damage melee thing I mentioned.

Reply:
Well farther down below where monk code I mean.

Reply:
There was a chunk of code I also couldn't add in at code/datums/martial.dm fromhttps://github.com/tgstation/tgstation/pull/35535 due to a cleanup of martial code at https://github.com/tgstation/tgstation/pull/46497 that could also been possible.

Reply:
Well I was right and with the help of the tg discord discord section the issue was found and fixed. Thanks 4dplanner.